front/src/common/Map/Search/useSearchOperationalPoint.tsx

```diff
@@ -39,9 +45,26 @@ export default function useSearchOperationalPoint({
           ['search', ['name'], debouncedSearchTerm],
           ['like', ['to_string', ['uic']], `%${debouncedSearchTerm}%`],
         ];
+
+    const dpyToMasOperationalpointsFilter =
+      isStdcm && hasOnlyStdcmRoles
```

Comment:
Can we check here that the user is not admin instead of checking that he has only StdcmRoles ? What if the user has Stdcm and OperationalStudies roles ?